Lib/enum.py

```diff
@@ -1396,7 +1396,7 @@ def global_flag_repr(self):
     module = self.__class__.__module__
     cls_name = self.__class__.__name__
     if self._name_ is None:
-        return "%x" % (module, cls_name, self._value_)
+        return "%s.%s(%x)" % (module, cls_name, self._value_)
```

Comment:
Do any of the unittests test this particular format codepath?  none of the added tests have reprs that end with `(0x8)` or similar.

Reply:
Yeah, no, I couldn't quite figure out how to make `_name_` to be None. When we hear back from Ethan, he'll hopefully suggest a test for this, too. Worst case we can just add a test using `repr(re.RegexFlag(0))`.

Reply:
https://bugs.python.org/msg399863